runtime/compiler/x/codegen/J9TreeEvaluator.hpp

```diff
@@ -151,6 +151,7 @@ class OMR_EXTENSIBLE TreeEvaluator: public J9::TreeEvaluator
    static TR::Register *awrtbarEvaluator(TR::Node *node, TR::CodeGenerator *cg);
    static TR::Register *awrtbariEvaluator(TR::Node *node, TR::CodeGenerator *cg);
    static TR::Register *inlineStringLatin1Inflate(TR::Node *node, TR::CodeGenerator *cg);
+   static TR::Register *inlineCRC32CUpdate(TR::Node *node, TR::CodeGenerator *cg);
```

Comment:
This should live in `x/amd64/codegen/J9TreeEvaluator.hpp` rather than the common evaluator.

The namespace on the implementation in the cpp file needs to be adjusted too.  Ideally, that implementation should live in a cpp file under amd64, but for some reason the precedent is to ifdef the implementation for 64-bit in the common file.  That's not ideal, and I'll create a separate issue for someone to fix.